### ISSUE TITLE
Calculator code

### ISSUE THREAD

**Issue by rrmcinty** [d-dtm/Kakuro-Helper#5]
The calculator code I added is totally broken, but I figured its better than nothing due to time constraints. I'm going to try and get it working by morning.

**Comment by rrmcinty** [comment]
Doesn't crash now, but still isn't doing what it's supposed to. Hopefully my comments help show what I intended on doing.

**Comment by d-dtm** [comment]
Was this pulled from his python code he sent us in an e-mail? That seems like a particularly useful resource.

**Comment by rrmcinty** [comment]
Yes, I intended on making it run the same. Since python lists don't need a set length, I decided to use ArrayLists in java to implement them in the same way.

**Comment by d-dtm** [comment]
Gotcha, keep in mind Java doesn't auto-clone lists. So for ArrayList<> x = for (i : ArrayList<> other){ x.append(i)} will add to the existing list, where the original python code he presented takes advantage of inherent cloning.

**Comment by rrmcinty** [comment]
Well dang, my understanding of all this is definitely not at the same level as you haha. I'm just going to try and help as best I can. I'm missing some classes today so I can figure some of this out since I have no time afterwards.

**Comment by d-dtm** [comment]
Well, I was looking for a stronger way to identify the locations for
upperNum/lower vals in the RuleCells and also paint a line between them,
and I saw that you had done that pretty well in your code if you want to
see about merging those aspects?

**Comment by rrmcinty** [comment]
The drawString doesn't work with a the Jtext field, so we can either not use drawstring or change the number values to ints and then to strings. The paint method is in the cell jpanel, so when we stretch the window I think it should adjust... but I uploaded the rulecell containing it.

**Comment by rrmcinty** [comment]
Nevermind it doesnt adjust. Maybe we can just keep the board at a fixed size instead of stretching with the window.
